src/pages/pass-finalisation/cat-be/components/code-78/code-78.ts

```diff
@@ -0,0 +1,77 @@
+import { Component, Input, OnChanges, Output, EventEmitter } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { TestCategory } from '../../../../../shared/models/test-category';
+import { Code78Present, Code78NotPresent } from '../../../../../modules/tests/pass-completion/pass-completion.actions';
+import { GearboxCategory } from '@dvsa/mes-test-schema/categories/Common';
+
+@Component({
+  selector: 'code-78',
+  templateUrl: 'code-78.html',
+})
+
+export class Code78Component implements OnChanges {
+  @Input()
+  form: FormGroup;
+
+  @Input()
+  category: String;
+
+  @Input()
+  transmission: GearboxCategory;
+
+  @Output()
+  code78Present = new EventEmitter<Code78Present>();
+
+  @Output()
+  code78NotPresent = new EventEmitter<Code78NotPresent>();
+
```

Comment:
I don't think rehydration will work on the component as your not passing the selected value back in and setting it